src/common/components/DataView.tsx

```diff
@@ -0,0 +1,60 @@
+import { DataObject } from '../types/NarrativeDoc';
+import { FC } from 'react';
+import { getWSTypeName } from '../utils/stringUtils';
+import classes from './DataView.module.scss';
+import TypeIcon from './TypeIcon';
+
+interface DataViewProps {
+  accessGroup: number;
+  dataObjects: DataObject[];
+}
+
+const DataView: FC<DataViewProps> = ({ accessGroup, dataObjects }) => {
+  if (!dataObjects?.length) {
+    return (
+      <p className={classes.dataview_no_data}>This narrative has no data.</p>
+    );
+  }
+
+  const rows = dataObjects
+    .slice(0, 50)
```

Comment:
Is there any indication to the user that this list gets truncated? Might be something to create a future ticket to address, if it isn't trivial to add.

Reply:
I think a future ticket would likely be the best scenario. This was the UI pulled verbatim from navigator. I agree that it should show some indication that it's not displaying all the objects but I'm not sure I have a design choice for that off the top of my head that would satisfy everyone. Since it likely involves further discussion I think it would be lest to leave as is in this particular PR